lib/todo/boards/board_lists/board_list_manager.ex

```diff
@@ -0,0 +1,85 @@
+defmodule Todo.Boards.BoardLists.BoardListManager do
+  alias Ecto.Multi
+  alias Todo.Boards.BoardList
+  alias Todo.Boards.BoardLists.BoardListPositions
+  alias Todo.Repo
+
+  @moduledoc """
+  This module handles creating and updating board lists. When a board list is created or updated,
+  the positions of the other board lists on that board will be re-ordered if required.
+  """
+
+  # @spec create(%{}, %Board{}) :: {:ok %BoardList{}} | {:error, String.t}
```

Comment:
I'm not sure why but I'm getting the this error for this spec
`(SyntaxError) syntax error before: '%'`

Reply:
`{:ok %BoardList{}} `

Missing a comma here between `ok` and `%`